setup.cfg

```diff
@@ -1,37 +1,39 @@
 [metadata]
 name = lcm
 description = This package aims to generalize and facilitate the specification, solving, and estimation of dynamic choice models.
-long_description = file: README.rst
-long_description_content_type = text/x-rst
-author = Janos Gabler, Hans-Martin von Gaudecker, Sebastian Gsell, Moritz Mendel, Christian Zimpelmann
-author_email = zimpelmann@iza.org
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/opensourceeconomics/lcm
+author = Janos Gabler, Tim Mensinger, Hans-Martin von Gaudecker
+author_email = hmgaudecker@uni-bonn.de
 license = MIT
 license_file = LICENSE
-platforms = unix, linux, osx, cygwin, win32
 classifiers =
     Development Status :: 1 - Planning
-    Environment :: Console
     Intended Audience :: Science/Research
     License :: OSI Approved :: MIT License
     Operating System :: MacOS :: MacOS X
-    Operating System :: Microsoft :: Windows
     Operating System :: POSIX
     Programming Language :: Python :: 3
     Programming Language :: Python :: 3 :: Only
-    Programming Language :: Python :: 3.6
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
     Topic :: Scientific/Engineering
-    Topic :: Software Development :: Build Tools
-project_urls =
-    Github = https://github.com/iza-institute-of-labor-economics/lcm
+keywords =
+    economics
+    econometrics
+    statistics
+    estimation
+    life cycle models
+    dynamic programming
 
 [options]
 packages = find:
 install_requires =
+    dags
     estimagic
-python_requires = >=3.6.1
+    jax
+    numpy
+    pandas
```

Comment:
Shouldn't this include everything from testenv? dags, networkx, pybaum, ...

Reply:
I'd say no. The testing process requires other packages, e.g., `scipy`, compared to the package itself. I've added comments to the environment files that should make this clear.

Reply:
Okay, maybe not everything, but anything that is imported in lcm proper, right?

Reply:
Yes, exactly. I think this should be the case now after my latest updates.